services/tasks/project/api/routes.py

```diff
@@ -125,42 +115,30 @@ def get_all_tasks():
 def populate_db():
     all_repos = current_app.config['REPOS']
 
-    response_object = dict()
+    repos = dict()
     for repo in all_repos:
         prs = get_items(repo['name'], 'pull_requests')
         issues = get_items(repo['name'], 'issues')
         items = issues + prs
-        response_object[repo['name']] = items
+        repos[repo['name']] = items
 
     for repo in all_repos:
-        if len(response_object[repo['name']]) != 0:
-            for issue in response_object[repo['name']]:
-                if issue != None:
-                    if 'pull' in issue['url']:
-                        task_type = 'pr'
-                    else:
-                        task_type = 'issue'
-                    creator = issue['author']
-                    url = issue['url']
-                    created_at = issue['createdAt']
-                    updated_at = issue['updatedAt']
-                    labels = issue['labels']
-                    num_of_comments = issue['comments']
-                    num_of_reactions = issue['reactions']
-                    title = issue['title']
-                    language = repo['programming_language']
+        if len(repos[repo['name']]) != 0:
+            for item in repos[repo['name']]:
+                if item != None:
+                    task_type = 'pr' if 'pull' in item['url'] else 'issue'
```

Comment:
Hopefully no repos have `pull` in their name (:

Reply:
Hmm, I think if I rename `repos` to `items` that should lessen the confusion.

Reply:
But the URL for an issue could have `pull` in it: `https://github.com/sendgrid/dx-pulley/issues/62`

Reply:
Ah, gotcha! I should look for `/pull/` instead :)